Documentation/contributing/development/dev_setup.rst

```diff
@@ -57,33 +57,39 @@ Version Requirements
++ `cfssl <https://github.com/cloudflare/cfssl>`_                    | >= v1.6.0                    | ``go install github.com/cloudflare/cfssl/cmd/cfssl@latest``     |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `cfssljson <https://github.com/cloudflare/cfssl>`_                | >= v1.6.0                    | ``go install github.com/cloudflare/cfssl/cmd/cfssljson@latest`` |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `Docker <https://docs.docker.com/engine/installation/>`_          | OS-Dependent                 | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `Docker-Compose <https://docs.docker.com/compose/install/>`_      | OS-Dependent                 | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ python3-pip                                                       | latest                       | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `helm <https://helm.sh/docs/intro/install/>`_                     | >= v3.6.0                    | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `kind <https://kind.sigs.k8s.io/docs/user/quick-start/>`_         | >= v0.7.0                    | ``go install sigs.k8s.io/kind@v0.19.0``                         |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`_      | >= v1.14.0                   | N/A (OS-specific)                                               |
++-------------------------------------------------------------------+------------------------------+-----------------------------------------------------------------+
++ `cilium-cli <https://github.com/cilium/cilium-cli#installation>`_ | Cilium-Dependent             | N/A (OS-specific)                                               |
```

Comment:
according to https://github.com/cilium/cilium-cli#releases, there is a dependency between cilium and cilium-cli, so i left it with `Cilium-Dependent` here.